examples/analytics.amp.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <style amp-custom>
+    amp-img {
+      max-width: 80%;
+      max-height: 80%;
+      width: 100%;
+    }
+  </style>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+<article>
+  <amp-analytics id="analytics1">
+  {
+    "host": "my-analytics.com",
+    "requests": {
+      "default": "/log?domain=DOMAIN&path=PATH"
```

Comment:
It will look like this:

```html
<amp-analytics id="analytics1">
  <script type="application/ld+json">
   {
    "host": "my-analytics.com",
    ...
  }
  </script>
</amp-analytics>
```